lib/Echidna/Exec.hs

```diff
@@ -70,12 +69,12 @@ execTxWith h m t = do (og :: VM) <- use hasLens
                       setupTx t
                       res <- m
                       cd  <- use $ hasLens . state . calldata
-                      case (res, isRight $ t ^. call) of
-                        (f@Reversion, _)         -> do hasLens .= og
-                                                       hasLens . state . calldata .= cd
-                                                       hasLens . result ?= f
-                        (VMFailure x, _)         -> h x
-                        (VMSuccess bc, True)     -> (hasLens %=) . execState $ do
+                      case (res, t ^. call) of
+                        (f@Reversion, _)            -> do hasLens .= og
+                                                          hasLens . state . calldata .= cd
+                                                          hasLens . result ?= f
+                        (VMFailure x, _)            -> h x
+                        (VMSuccess bc, SolCreate _) -> (hasLens %=) . execState $ do
```

Comment:
I know it's been like that before but is this case exhaustive?

Reply:
the next case is a catch-all

Reply:
thanks, I've missed that 👍